index.rst

```diff
@@ -147,176 +246,121 @@ The white light source and monochromator sit on a table surrounded by sturdy bla
-#. Install fiber spectrograph
-
-    .. figure:: /_static/spectro.png
-       :name: spectro
-       :target: ../_images/spectro.png
-       :alt: spectro
-       :scale: 15 %
-
-#. Setup optics
-
-    .. figure:: /_static/optics_top.png
-       :name: optics_top
-       :target: ../_images/optics_top.png
-       :alt: optics_top
-       :scale: 15 %
-
-#. Install glass window in shutter
-#. Assemble shutter
-#. Install limit switches
-
-    .. figure:: /_static/open_shutter.png
-       :name: open_shutter
-       :target: ../_images/open_shutter.png
-       :alt: open_shutter
-       :scale: 15 %
-
-    .. figure:: /_static/shutter_open_2.png
-       :name: shutter_open2
-       :target: ../_images/shutter_open_2.png
-       :alt: shutter_open2
-       :scale: 15 %
-
-    .. figure:: /_static/shutter_close.png
-       :name: shutter_close
-       :target: ../_images/shutter_close.png
-       :alt: shutter_close
-       :scale: 15 %
-
-    .. figure:: /_static/shutter_close_2.png
-       :name: shutter_close2
-       :target: ../_images/shutter_close_2.png
-       :alt: shutter_close2
-       :scale: 10 %
+      .. figure:: /_static/step_2.png
+         :name: step_2
+         :target: ../_images/step_2.png
+         :alt: step_2
+
+#. Remove the linear actuator and place it out of the way
 
-#. Open slits on monochromator
-#. Confirm that remote control is selected on WLS
+      .. figure:: /_static/step_3.png
+         :name: step_3
+         :target: ../_images/step_3.png
+         :alt: step_3
 
-Now you are ready to start turning things on and testing things. 
+#. Remove the final bolt on the shutter and remove the shutter
 
-Operational Setup
------------------
-First, addresses for all of the control components will have to be established. That is being tracked in JIRA ticket `IT-4096 <https://jira.lsstcorp.org/browse/IT-4096>`__. 
+      .. figure:: /_static/step_4.png
+         :name: step_4
+         :target: ../_images/step_4.png
+         :alt: step_4
 
-Make sure that the indexes for each item is correct (for the CSC) and override yaml files are defined and identified.
+#. If you need to open the center door, remove the "closed" limit switch
 
-Then, go to the address for the PDU and confirm that the numbering of the outlets agrees with the table above and what is actually implemented.
+      .. figure:: /_static/step_5.png
+         :name: step_5
+         :target: ../_images/step_5.png
+         :scale: 50 %
 
-You can now power everything up. Initialize the monochromator by logging into the windows machine via remote desktop.
+
+Chiller Maintenance
+-------------------
+
+You may need to refill the water in the chiller occassionally. This should be done with distilled water mixed with 10% of methanol.
+
+.. figure:: /_static/fill_cover.png
+   :name: fill_cover
+   :target: ../_images/fill_cover.png
+   :alt: fill_cover
+   :scale: 50 %
+
+   Chiller cap for refilling water/methanol mixture.
+
+Using Fiber Illuminator 
```

Comment:
Maybe just "Fiber Illuminator"
or  "Using the Fiber Illuminator' ?

Reply:
done